### PR TITLE
chore: switched development version to v20

### PR DESCRIPTION
**Pull request by kirrg001**
refs https://jsw.ibm.com/browse/INSTA-983

Now that we have [resolved the issue with ESM ](https://github.com/instana/nodejs/pull/1063)and Node v18.18/v18.19, we can switch to Node v20.
Node v20 is LTS.

- [x] switch pipelines to use v20
- [x] manually verify that the docker images build
- [x] fixed slack notifications for ci.instana.io
- [x] add global "node-version" in Tekton which is always used when the pipeline itself did not overwrite it (Tekton automatically sends the correct version)

### CODE REVIEW

File: packages/azure-container-services/package.json [hunk 1]
```diff
@@ -61,6 +61,9 @@
   "bugs": {
     "url": "https://github.com/instana/nodejs/issues"
   },
+  "engines": {
+    "node": ">=14.0.0"
+  },
```
Comment (kirrg001) [inline packages/azure-container-services/package.json:66]: @aryamohanan Engine definition was missing for this package. Do you have any idea why? Is Node v14 correct for azure container services? Thanks

Comment (aryamohanan) [reply packages/azure-container-services/package.json:66]: If I recall correctly, the reason is that when we added the support azure-app-services, the supported Node.js versions were 16, 18, and 20. You can refer to the comment here: [GitHub](https://github.com/instana/nodejs/blob/main/packages/azure-container-services/src/index.js#L9). Additionally, we can verify the supported versions in the official documentation: [Microsoft documentation](https://learn.microsoft.com/en-us/azure/app-service/configure-language-nodejs?pivots=platform-linux).

Comment (kirrg001) [reply packages/azure-container-services/package.json:66]: I saw that the [test](https://github.com/instana/nodejs/blob/v3.8.1/packages/azure-container-services/test/using_api/test.js) is not skipped for < 16. Thats why I got confused :) 
Should I use "node": ">=14.0.0" or "node": ">=16.0.0"?

Comment (aryamohanan) [reply packages/azure-container-services/package.json:66]: We are not calling the actual Azure app service in the tests, same as GCP.

Comment (kirrg001) [reply packages/azure-container-services/package.json:66]: Then I keep 

> "node": ">=14.0.0"

File: packages/serverless/ci/pipeline.yaml [hunk 2]
```diff
@@ -108,7 +108,7 @@ resources:
     type: slack-notification
     icon: slack
     source:
-      url: ((tracer-community-slack-hook))
+      url: ((team-nodejs-slack-hook))
```
Comment (kirrg001) [inline packages/serverless/ci/pipeline.yaml:111]: Added a new nodejs slack webhook.

**Comment by kirrg001** [comment]
I have already started to test the images. PR is RFR.

File: packages/serverless/ci/testing/pipeline-debug.yaml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      username: ((instanacd-github-api-token))
+      password: x-oauth-basic
+
+
+  - name: instana-aws-fargate-npm-package
+    type: npm-resource
+    icon: cube-outline
+    source:
+      scope: instana
+      package: aws-fargate
```
Comment (aryamohanan) [inline packages/serverless/ci/testing/pipeline-debug.yaml:20]: Is this specific to aws-fargate? As its for local testing we can name it generic?

File: packages/aws-fargate/images/inspector/Dockerfile [hunk 4]
```diff
@@ -1,4 +1,6 @@
-FROM node:18-alpine
+ARG NODEJS_VERSION
```
Comment (aryamohanan) [inline packages/aws-fargate/images/inspector/Dockerfile:1]: qs: From where this value is set?

File: packages/serverless/ci/testing/pipeline-debug.yaml (hunk 3 above)
Comment (kirrg001) [reply packages/serverless/ci/testing/pipeline-debug.yaml:20]: This resource downloads the fargate npm package. The testing pipeline is just a minified version of the real serverless pipeline. We can't rename this.

File: packages/aws-fargate/images/inspector/Dockerfile (hunk 4 above)
Comment (kirrg001) [reply packages/aws-fargate/images/inspector/Dockerfile:1]: https://github.com/instana/nodejs/pull/1152/files/15ed7d909f067d2e6e6cb507d3fd7f9ca256b9e5#diff-02f198aef71805a77d9836533174d3e061195b4560547217b8fcad4978a8bbbb

**Review by aryamohanan — APPROVED** [review]

**Pull request merged**